WcaOnRails/app/views/users/index.html.erb

```diff
@@ -1,11 +1,23 @@
 <% provide(:title, 'Users') %>
 
 <div class="container">
-  <h1><%= yield(:title) %></h1>
+  <h2><%= yield(:title) %></h2>
+
+  <div id="users-query-fields" class="form-inline bs-table-query-fields">
+    <div id="search-box" class="form-group">
+      <div class="input-group">
+        <%= content_tag :span, icon("search"), class: "input-group-addon",
+                               data: { toggle: "tooltip", placement: "top" },
+                               title: "Type name, WCA ID, email or country. Use a space to separate them." %>
```

Comment:
nit: I believe we have been using the [Oxford comma](https://en.wikipedia.org/wiki/Serial_comma) in [other places](https://github.com/jfly/worldcubeassociation.org/blob/777e31466272259162a3870a651656234d1455d1/WcaOnRails/config/locales/en.yml#L417).